locations/spiders/lidl.py

```
@@ -0,0 +1,113 @@
+            features = [x["name"] for x in location["marketingData"]["infoIcons"]]
+            apply_yes_no("cash_withdrawal", item, "cashBack" in features)
+            apply_yes_no(Extras.TOILETS, item, "customerToilet" in features)
+            apply_yes_no(Extras.BABY_CHANGING_TABLE, item, "diaperChangingStation" in features)
+            apply_yes_no(Extras.WIFI, item, "freeWiFi" in features)
+            apply_yes_no(Extras.SELF_CHECKOUT, item, "shopNGo" in features)
+            apply_yes_no(Extras.WHEELCHAIR, item, "wheelchair" in features)
+
+            try:
+                item["opening_hours"] = self.parse_opening_hours(location["openingHours"])
+            except Exception:
+                pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)